.travis/metaSMT.sh

```diff
@@ -6,6 +6,7 @@ sudo apt-get -y install libboost1.55-dev libz3 libz3-dev libgmp-dev
 # Clone
 git clone git://github.com/hoangmle/metaSMT.git
 cd metaSMT
+git checkout 1f9e399
```

Comment:
Like we do with STP version. I think it would be better to have the version as an environment variable `METASMT_VERSION` (see https://github.com/klee/klee/blob/master/.travis/stp.sh) and then set that environment variable for the the appropriate build configurations in `.travis.yml`.